scripts/pso-collect-logs.sh

```diff
@@ -55,7 +59,7 @@ done
 echo "collect info for all pods"
 $KUBECTL get pod --all-namespaces -o wide > $LOG_DIR/all-pods.log
 echo -e "\n" >> $LOG_DIR/all-pods.log
-$KUBECTL describe pod -n $PSO_NS >> $LOG_DIR/all-pods.log
+$KUBECTL describe pod --all-namespaces >> $LOG_DIR/all-pods.log
```

Comment:
Do we want to collect pods info outside of PSO?

Reply:
I assume this is so we can see if the CNI is running correctly. It might be better to break this into two calls. One that gets all the PSO_NS pods and one that gets the k8s system pods in the `kube-system` namespace. This way we don't get customer application info.

Reply:
This is also to collect pod that mounts the PVC, if mount fails we want to see it. hence not just pso namespace and kube-system namespace. @dsupure @sdodsley

Reply:
What we say is we shouldn't collect customer pods info outside of pso and kube-system namespaces. Customers may have some sensitive info in those pods.

Reply:
i am not too concerned about it, customer pod is only app name, we dont collect app data or app secret.

Reply:
@sdodsley what do you think? if we concern about collecting pod info in user namespace, then pvcs/pvs and kube-system namespace may be in question too.

Reply:
That is a good point. We won't get some of the info we need.
Maybe we have 2 modes. One that only collects the PSO and kube-system namespace pods. The second, maybe use a `-full` option on the script to collect ALL pods in all namespaces.
This gives the customer the option.

Reply:
@haibinxie @dsupure what are your thoughts on using a `-full` option?

Reply:
I think full option is good. shall we popup for users to choose instead of parameter -- which is often forgotten?

Reply:
I would leave the minimal as the default / announced option and we only tell customers about the `--full` switch if necessary

Reply:
sounds good to me!